### DISCUSSION TITLE
Display gets extremely hot

### DISCUSSION THREAD

**Discussion by emaddeldin** [MonitorControl/MonitorControl#1386] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Question

The display gets extremely warm when using the extended XDR display, how does this affect it's longevity ?

**Comment by waydabber** [comment]
I am not sure I understand the problem. What do you mean by "extended XDR display" in the context of MonitorControl? Are you referring to XDR brightness upscaling in BetterDisplay? If so, take a look at this: https://github.com/waydabber/BetterDisplay/wiki/XDR-and-HDR-brightness-upscaling#softwarehardware-safety-considerations
